portfolio/src/main/webapp/script.js

```diff
@@ -17,7 +17,8 @@
  */
 function addFunFact() {
   const fun_facts =
-      ['My favorite color is purple.', 'I have a little brother.', 'I am half Chinese.', 'I am 18 years old.'];
+      ['My favorite color is purple.', 'I have a little brother.', 'I am half-Chinese.', 'I am 18 years old.', 'My elbows are double-jointed.',
```

Comment:
Recommend putting each element of the array on a new line. 

General guidance: If the elements all fit on one line, then one line is fine. However, if they wrap over multiple lines, then each element on its own line is preferred.

Reply:
Done! (see commit "Address requested changes...")